es8.md

```diff
@@ -1,10 +1,10 @@
-# [Async Functions](https://github.com/lukehoban/ecmascript-asyncawait)
+This document specifies the extensions to the core ESTree AST types to support the ES8/ES2017 grammar.
 
 ## Function
 
 ```js
 extend interface Function {
-    async: false;
+    async: boolean;
```

Comment:
We have the same in babylon https://github.com/babel/babylon/blob/master/ast/spec.md#functions - it makes sense to do since the other booleans are just `boolean` and I guess false was originally there to signify a default